extensions/gulp/package.json

```diff
@@ -6,7 +6,7 @@
   "version": "1.0.0",
   "icon": "images/gulp.png",
   "engines": {
-    "vscode": "*"
+    "vscode": "0.10.x"
```

Comment:
Why do we need to change this. Usually we mark prebundled extension with '*' since they live in our source tree?

Reply:
I was just copying what another extension did. I don't think we need this.